test/jdk/java/nio/file/Path/MacPathTest.java

```diff
@@ -22,7 +22,7 @@
  */
 
 /* @test
- * @bug 7130915 8289689
+ * @bug 7130915 8289689 8366911
```

Comment:
This test has `@run main/othervm -Djdk.nio.path.useNormalizationFormD=true` that I assume should be removed.